cypress/utils/utils.ts

```diff
@@ -1858,7 +1858,7 @@ export function validateTackleCr(): void {
 
 export function validateMtaOperatorLog(): void {
     cy.wait(30 * SEC);
-    let command = `oc logs $(oc get pods | grep mta-operator | cut -d " " -f 1) | grep failed | tail -n 1| awk -F 'failed=' '{print $2}'|cut -d " " -f 1`;
+    let command = `oc logs $(oc get pods -n openshift-mta | grep mta-operator | cut -d " " -f 1) -n openshift-mta | grep failed | tail -n 1| awk -F 'failed=' '{print $2}'|cut -d " " -f 1`;
```

Comment:
+1. This is important cause it can be installed in another namespace. But it looks strange for me that NS is required cause it might go to NS in previous commands

Reply:
I'm surprised as well because the existing test passed with the above command at one point  .